getting-started/src/main/kotlin/com/google/cloud/kotlin/demo/EmojifyController.kt

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.kotlin.demo
+
+import com.google.cloud.storage.Acl
+import com.google.cloud.storage.Storage
+import com.google.cloud.storage.StorageOptions
+import com.google.cloud.vision.v1.*
+import com.google.cloud.vision.v1.Feature.Type
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import java.awt.Polygon
+import java.io.FileOutputStream
+import java.io.PrintStream
+import java.nio.file.Files
+import java.nio.file.Paths
+import javax.imageio.ImageIO
+
+val storage: Storage = StorageOptions.getDefaultInstance().service
+val bucket = storage.get("cloud-kotlin-samples")
+val bucketUri = "gs://cloud-kotlin-samples"
```

Comment:
This variable is unnecessary, let's just prepend `gs://` when we need the URI (see comment below)